operator/watchers/k8s_service_sync.go

```diff
@@ -48,6 +48,7 @@ var (
 	// k8s.
 	k8sSvcCacheSynced = make(chan struct{})
 	kvs               *store.SharedStore
+	sharedOnly        bool
```

Comment:
Could you describe why this is needed and what it stands for?

Reply:
Operator use (the original use) syncs only shared services. Clustermesh-apiserver needs to sync all services so that the VMs get access to them.

Reply:
Could you add that to src?

Reply:
Done.